mkdocs/docs/configuration.md

```diff
@@ -95,6 +95,7 @@ Iceberg works with the concept of a FileIO which is a pluggable module for readi
 - **hdfs**: `PyArrowFileIO`
 - **abfs**, **abfss**: `FsspecFileIO`
 - **oss**: `PyArrowFileIO`
+- **hf**: `FsspecFileIO`
```

Comment:
is there a way to allow `PyArrowFileIO` as well?

Reply:
There is no HF filesystem implementation in arrow C++ yet unfortunately ! But hopefully soon